test/test_torch.py

```diff
@@ -2757,6 +2757,18 @@ def test_pickle_function(self):
             b = pickle.loads(serialized)
             self.assertEqual(a, b)
 
+        def test_norm_deprecated(self):
```

Comment:
Nit: this and the next test should probably be put into test_linalg.py

Reply:
Done. I also changed the fastpath test to call `torch.linalg.norm` instead of `torch.norm`.